### PR TITLE
missing cross origin setting

### PR DESCRIPTION
**Pull request by DavidHGillen**
necessary for the https hosting used on playground demos and many other places.

### CODE REVIEW

File: src/Helpers/babylon.videoDome.ts [hunk 1]
```diff
@@ -75,6 +75,7 @@ module BABYLON {
             // configure material
             texture.coordinatesMode = BABYLON.Texture.FIXED_EQUIRECTANGULAR_MIRRORED_MODE; // matches orientation
             texture.wrapV = Texture.CLAMP_ADDRESSMODE; // always clamp the up/down
+            texture.video.crossOrigin = "anonymous";
```
Comment (deltakosh) [inline src/Helpers/babylon.videoDome.ts]: Please use texture.video.crossOrigin =Tools.CorsBehavior
Also I would prefer to update the videoTexture object directly

Comment (DavidHGillen) [reply src/Helpers/babylon.videoDome.ts]: I'll make the first change but am unsure what you mean by the second.
"Also I would prefer to update the videoTexture object directly"

**Comment by deltakosh** [comment]
I meant: we should do that directly inside the video texture class when instantiating the video element

**Comment by DavidHGillen** [comment]
As an upgrade to VideoTexture itself, rather than specifically for VideoDome? I can make that change for you.

**Comment by deltakosh** [comment]
Yes please

File: src/Materials/Textures/babylon.videoTexture.ts [hunk 2]
```diff
@@ -84,6 +84,20 @@
             }
         }
 
+        private _setCORS(src: string, element: HTMLVideoElement): void {
```
Comment (deltakosh) [inline src/Materials/Textures/babylon.videoTexture.ts]: You can use Tools.SetCordBehavior for that

File: src/Materials/Textures/babylon.videoTexture.ts [hunk 3]
```diff
@@ -92,18 +92,21 @@
...
         };
     
         private _getVideo(src: string | string[] | HTMLVideoElement): HTMLVideoElement {
             if (src instanceof HTMLVideoElement) {
+                Tools.SetCorsBehavior(src.currentSrc, src);
                 return src;
             }
             const video: HTMLVideoElement = document.createElement("video");
             if (typeof src === "string") {
+				Tools.SetCorsBehavior(src, video);
```
Comment (deltakosh) [inline src/Materials/Textures/babylon.videoTexture.ts]: tabbing seems off

Comment (DavidHGillen) [reply src/Materials/Textures/babylon.videoTexture.ts]: sorry about that, just updated webstorm and apparently it decided this document full of spaces should use tabs for it's tabbing on those lines. Used to behave better than that, so I forgot to check. I'll fix this a quick second.

**Comment by deltakosh** [comment]
Thanks I'll create a nightly to see if this fix the PG

**Comment by deltakosh** [comment]
This seems not to work : https://www.babylonjs-playground.com/#1E9JQ8#2

**Comment by DavidHGillen** [comment]
It has corrected the console errors, the playground environment is the only change when compared to my working versions (it works locally and hosted on my servers). I'll do what I can to track it down.

**Comment by deltakosh** [comment]
Does it work in the local playground (when using the playground debug mode defined here: http://doc.babylonjs.com/how_to/how_to_start#debug) ?

**Comment by DavidHGillen** [comment]
Figured out the issue, the camera is silently failing. Fixed version with
Free https://www.babylonjs-playground.com/#1E9JQ8#3
Arc https://www.babylonjs-playground.com/#1E9JQ8#4
any preference as to which one is pushed?

**Comment by deltakosh** [comment]
Arc is perfect!

**Pull request merged**